azure-iot-device/azure/iot/device/common/transport/pipeline_ops_base.py

```diff
@@ -162,3 +162,81 @@ def __init__(self, sas_token, callback=None):
         """
         super(SetSasToken, self).__init__(callback=callback)
         self.sas_token = sas_token
+
+
+class IotRequestAndResponse(PipelineOperation):
```

Comment:
>IotRequestAndResponse [](start = 6, length = 21)

the name of this does not feel like an operation. It feels more like a model or object name.
Not sure what to call it but it is lacking some verb..